src/main/java/org/broadinstitute/hellbender/tools/funcotator/DataSourceFuncotationFactory.java

```diff
@@ -217,9 +224,27 @@ private boolean isFeatureListCompatible(final List<Feature> featureList) {
      * @param featureContext the FeatureContext to query
      * @return Features from our FeatureInput {@link #mainSourceFileAsFeatureInput} queried from the FeatureContext
      */
+    @SuppressWarnings("unchecked")
```

Comment:
Any idea of the impact on performance here?

Reply:
There was a `@SuppressWarnings...` block over a method here already, so I don't think it will affect performance.  

Does SuppressWarnings do anything at runtime or is it just a compiler check?